doc/Setup.md

```diff
@@ -21,7 +21,35 @@ $> eups distrib install lsst_sims -t sims
 $> setup lsst_sims -t sims
 ```
 
-2) Install `PhoSim`.
+2) Enable connection to CatSim databases
```

Comment:
Maybe add a note here (with a link to `Sims_phoSimGen.md`) that for whitelisted machines, the procedure is different?